cmd/run.php

```diff
@@ -3,41 +3,34 @@
 declare(strict_types=1);
 
 use GuzzleHttp\Client;
-use GuzzleHttp\Exception\GuzzleException;
-use Interns2022B\GithubScrapper;
+use Interns2022B\BreweryScrapper;
 use League\CLImate\CLImate;
 
 require_once __DIR__ . "/../vendor/autoload.php";
 
 $climate = new CLImate();
-
 $climate->green("Wow! My first simple PHP CLI application works!");
-$name = $climate->input("Please type a Github organization name:")->prompt();
 
-try {
-    $scrapper = new GithubScrapper($name, new Client());
-} catch (GuzzleException) {
-    $climate->red("Requested organization does not exist!");
-    die();
+$breweryName = $climate->input("Please provide brewery or city name:")->prompt();
```

Comment:
I would move this action to be one of the options in the menu. Right now it's a first thing that displays after the program starts.

Reply:
changed